src/main/java/com/bigt/simmet/models/dtos/ServiceDto.java

```diff
@@ -0,0 +1,21 @@
+package com.bigt.simmet.models.dtos;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
What does this do?

Reply:
This automatically generates getters and setters for the DTO class